### PR TITLE
Add RF kill switch button for Zbook 15 G2

### PR DESCRIPTION
**Pull request by CedricDViou**
Seems to be the same KEYBOARD_KEY_ as "HP ZBook Studio G4".
Could they be merged?

### CODE REVIEW

File: hwdb/60-keyboard.hwdb [hunk 1]
```diff
@@ -577,7 +577,8 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHP*:pnHP*ProBook*4*:pvr*
 evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHewlett-Packard*:pnHPZBook*:pvr*
 evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHP*:pnHPZBook*:pvr*
  KEYBOARD_KEY_81=f20                                    # Fn+F8; Microphone mute button, should be micmute
-
+ KEYBOARD_KEY_f8=wlan                                   # Wireless HW switch button
+
```
Comment (yuwata) [inline hwdb/60-keyboard.hwdb]: Unnecessary white spaces in this blank line. This seems to cause the CI failures.

Comment (CedricDViou) [reply hwdb/60-keyboard.hwdb]: You are right.
How should I proceed? revert, commit, and pull request again? Or something more clever?
It's the first time to do a such thing... Sorry

Comment (yuwata) [reply hwdb/60-keyboard.hwdb]: Force-push is better. But in this case, adding a commit to remove the whitespace is also ok.

**Comment by CedricDViou** [comment]
> I'm not sure if we should add this definition to all those models. It seems safe to split out the pattern for `pnHPZBook*` and add the new definition only there.

I'm fine with that.  I wanted to make sure that I was not duplicating a definition.

**Comment by keszybz** [comment]
@CedricDViou please adjust your patch like that then.

**Pull request merged**